dotCMS/src/integration-test/java/com/dotmarketing/db/SystemEnvDataSourceStrategyTest.java

```diff
@@ -0,0 +1,93 @@
+package com.dotmarketing.db;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+import com.dotcms.util.IntegrationTestInitService;
+import com.dotcms.util.TestInitialContext;
+import com.dotmarketing.util.Constants;
+import com.liferay.util.SystemEnvironmentProperties;
+import com.zaxxer.hikari.HikariDataSource;
+import java.sql.SQLException;
+import java.util.HashMap;
+import java.util.Map;
+import javax.naming.NamingException;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+/**
+ * Tests for {@link SystemEnvDataSourceStrategy}
+ * @author nollymar
+ */
+public class SystemEnvDataSourceStrategyTest {
+
+    @BeforeClass
+    public static void prepare() throws Exception {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5026767)